File: consumer/internal/buildinfo/buildinfo.go
// SPDX-License-Identifier: MIT
// Package buildinfo holds linker-injected build metadata.
package buildinfo

// Set at link time via -ldflags; defaults suit local `go run`.
var (
	Version   = "dev"
	GitSHA    = "unknown"
	BuildTime = "unknown"
)

File: ingestion/src/build_info.rs
// SPDX-License-Identifier: MIT
//! Compile-time build metadata (see `build.rs`).

/// Crate version from `Cargo.toml`.
pub const VERSION: &str = env!("CARGO_PKG_VERSION");

/// Short git SHA (or `unknown` when not built from a git checkout).
pub const GIT_SHA: &str = env!("GIT_SHA");

/// UTC build timestamp (RFC3339).
pub const BUILD_TIME: &str = env!("BUILD_TIME");
